client/homebrew/editor/tagInput/tagInput.jsx

```diff
@@ -0,0 +1,105 @@
+require('./tagInput.less');
+const React = require('react');
+const { useState, useEffect } = React;
+const _     = require('lodash');
+
+const TagInput = ({ unique = true, values = [], ...props }) => {
+	const [temporaryValue, setTemporaryValue] = useState('');
+	const [valueContext, setValueContext] = useState(values.map((value) => ({ value, editing: false })));
```

Comment:
I'm also not sure on this name. This refers to the current list of tags, right? Why not `tagList` or `tagArray` or something?

Reply:
It’s a holdover from the original string array editor. I can change it no problem